sys/kern/kern_exit.c

```diff
@@ -205,12 +205,13 @@ exit_onexit(struct proc *p)
 /*
  * exit -- death of process.
  */
-void
-sys_sys_exit(struct thread *td, struct sys_exit_args *uap)
+int
+sys_exit(struct thread *td, struct exit_args *uap)
 {
 
 	exit1(td, uap->rval, 0);
 	/* NOTREACHED */
+	__unreachable();
```

Comment:
exit1() is declared with __dead2, so this should be excessive.  I think the lint NOTREACHED annotation is useless as well.

Reply:
clang objected to the lack of a return(some_int) without this addition. I'd debated with myself about the NOTREACHED and will remove it.

Reply:
We removed lint(1) from the system years ago, and these markers were for lint(1) and a few other older / ancient analysis tools that are no longer relevant. I see no reason to keep it, honestly, except fear of breaking something unknown... At this point, i can't even guess what that might be.